### DISCUSSION TITLE
Not updating my PR automatically even though automerge is enabled.

### DISCUSSION THREAD

**Discussion by anburocky3** [renovatebot/renovate#18032] [Request Help] [answered]
These are my config: 

**What i want?**
 Merge the deps automatically without human intervention if there is no conflict.

For example check this pr: https://github.com/anburocky3/CyberDude-Updates/pull/21

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":disableDependencyDashboard",
    ":preserveSemverRanges"
  ],
  "packageRules": [
    {
      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
      "matchDepTypes": ["devDependencies"],
      "automerge": true
    }
  ],
  "platformAutomerge": true,
  "commitBody": "[skip ci]"
}
```

> Note: I don't have any tests write now, just an simple react app.

**Comment by viceice** [comment, marked as answer]
you need some CI checks for automerge.

this can be disabled but we strongly discourage that

**Reply by rarkins** [reply]
`ignoreTests` (use at own risk)

**Reply by anburocky3** [reply]
Thank you, Currently, i don't have tests in my CI. should implement in the future, for now, it should be good to go. Thanks.

Do i need both, 
```
 "automerge": true
 "platformAutomerge": true,
 ```
 
 What exactly is platformAutomerge?

**Reply by rarkins** [reply]
platformAutomerge means to use GitHub, GitLab or Azure's own automerge features. e.g. for GitHub it is https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request

But they perhaps won't work without tests, and won't offer an override like we do.

If you leave with automerge=true then Renovate will automerge PRs itself.
